pages/src/TodoList.js

```diff
@@ -5,8 +5,13 @@ import { createSelector } from 'reselect'
 import { injectState, provideState, update, mergeIntoState } from "freactal";
 import { Map } from 'immutable'
 
+const persistEventInputValue = fn => (event, ...args) => fn(event.target.value, ...args)
```

Comment:
`persistEventInputValue` → `extractEventValue`.

Reply:
Got it :) next time, I will choose carefuly the names of my methods